docs/walkthrough-1.6.md

```diff
@@ -1,148 +1,21 @@
-# Service Catalog Demonstration Walkthrough
+# Service Catalog Demonstration Walkthrough (DEPRECATED)
 
 This document outlines the basic features of the service catalog by walking
 through a short demo.
 
-## Step 0 - Prerequisites
+This document contains instructions for a self-guided demo of the Service 
```

Comment:
Might want to put a bold NOTE here or something.

Reply:
done